### PR TITLE
Testv0.2

### PR DESCRIPTION
**Pull request by hakymulla**

### CODE REVIEW

File: starknet/src/L1_headers_store/contract.cairo [hunk 1]
```diff
@@ -12,12 +12,14 @@ pub mod L1HeaderStore {
     use fossil::library::blockheader_rlp_extractor::{
         decode_parent_hash, decode_uncle_hash, decode_beneficiary, decode_state_root,
         decode_transactions_root, decode_receipts_root, decode_difficulty, decode_base_fee,
-        decode_timestamp, decode_gas_used
+        decode_timestamp, decode_gas_used, decode_block_number
     };
+    use fossil::library::keccak256::keccak256;
```
Comment (ametel01) [inline starknet/src/L1_headers_store/contract.cairo]: Here we can group the imports to use less lines, eg: all the imports from `library` with only one use, same for the others

File: starknet/src/testing/proofs/storage.cairo [hunk 2]
```diff
@@ -1027,3 +1027,37 @@ pub fn PROOF_2() -> StorageProof {
         value: 0x07a8f1298f7eaa479401,
     }
 }
+
+pub fn PROOF_anvil() -> StorageProof {
```
Comment (ametel01) [inline starknet/src/testing/proofs/storage.cairo]: Are this proofs actually taken from anvil?

**Review by ametel01 — CHANGES_REQUESTED** [review]

File: starknet/src/testing/proofs/storage.cairo (hunk 2 above)
Comment (hakymulla) [reply starknet/src/testing/proofs/storage.cairo]: yeah, ran `anvil`, did some transactions, got the `Computed Block Hash` and `rlp` using `eth-rlp-verify` (which is different from the block hash shown in anvil) and the `mmr proof` using `headers-accumulators`.

Comment (ametel01) [reply starknet/src/testing/proofs/storage.cairo]: What about the stamp rate proofs? Are those the one la from the ipynb or are also from Anvil?

Comment (hakymulla) [reply starknet/src/testing/proofs/storage.cairo]: storage proof?

**Pull request closed without merging**